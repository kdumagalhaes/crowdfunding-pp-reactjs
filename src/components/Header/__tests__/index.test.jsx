import { render } from "@testing-library/react";
import { Header } from "..";

describe("Header", () => {
  it("initial component should match snapshot", () => {
    const { asFragment } = render(<Header />);
    expect(asFragment()).toMatchSnapshot();
  });
});
