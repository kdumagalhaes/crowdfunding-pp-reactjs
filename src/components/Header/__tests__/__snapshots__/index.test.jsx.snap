// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header initial component should match snapshot 1`] = `
<DocumentFragment>
  <header
    class="sc-bdvvtL gzsorK"
  >
    <div
      class="gradient"
    />
    <div
      class="sc-gsDKAQ hwssex"
    >
      <a
        href="#"
      >
        <h1
          class="seo-h1"
        >
          Crowdfunding • This is just for SEO best practices =)
        </h1>
        <img
          alt="crowdfund"
          src="logo.svg"
        />
      </a>
      <nav
        class="sc-dkPtRN kyRLwV"
      >
        <button
          class="hamburger-menu"
        >
          <img
            alt="open menu"
            src="icon-hamburger.svg"
          />
        </button>
        <ul>
          <li>
            <a
              href="#about"
            >
              About
            </a>
          </li>
          <li>
            <a
              href="#discover"
            >
              Discover
            </a>
          </li>
          <li>
            <a
              href="#get-started"
            >
              Get Started
            </a>
          </li>
        </ul>
      </nav>
    </div>
  </header>
</DocumentFragment>
`;

exports[`Header renders like expected 1`] = `
<DocumentFragment>
  <header
    class="sc-bdvvtL gzsorK"
  >
    <div
      class="gradient"
    />
    <div
      class="sc-gsDKAQ hwssex"
    >
      <a
        href="#"
      >
        <h1
          class="seo-h1"
        >
          Crowdfunding • This is just for SEO best practices =)
        </h1>
        <img
          alt="crowdfund"
          src="logo.svg"
        />
      </a>
      <nav
        class="sc-dkPtRN kyRLwV"
      >
        <button
          class="hamburger-menu"
        >
          <img
            alt="open menu"
            src="icon-hamburger.svg"
          />
        </button>
        <ul>
          <li>
            <a
              href="#about"
            >
              About
            </a>
          </li>
          <li>
            <a
              href="#discover"
            >
              Discover
            </a>
          </li>
          <li>
            <a
              href="#get-started"
            >
              Get Started
            </a>
          </li>
        </ul>
      </nav>
    </div>
  </header>
</DocumentFragment>
`;
